packages/cli/src/util/projects/project-settings.ts

```diff
@@ -51,3 +53,26 @@ export async function writeProjectSettings(
 export async function readProjectSettings(cwd: string) {
   return await getLinkFromDir<ProjectLinkAndSettings>(cwd);
 }
+
+export function pickOverrides(
+  vercelConfig: VercelConfig
+): PartialProjectSettings {
+  const overrides: PartialProjectSettings = {};
+  for (const prop of [
```

Comment:
I've seen similar code like this in the API and now in the CLI. In a future PR, we should consider consolidating some of these into helpers so we can add new overrides in the future.